src/FxCop/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -118,48 +118,36 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="DoNotCatchCorruptedStateExceptions" xml:space="preserve">
-    <value>Do not catch corrupted state exceptions in general handlers.</value>
+    <value>Do Not Catch Corrupted State Exceptions</value>
   </data>
   <data name="DoNotCatchCorruptedStateExceptionsDescription" xml:space="preserve">
     <value>Do not author general catch handlers in code that receives corrupted state exceptions. Code that receives and intends to handle corrupted state exceptions should author distinct handlers for each exception type.</value>
   </data>
   <data name="DoNotCatchCorruptedStateExceptionsMessage" xml:space="preserve">
```

Comment:
Messages should be small - they need to be able to show up in a single. You can put a lot of this detail in the description.I would change the message to be simply  "{0} is catching a corrupted state exception.". 
The description can be: "Catching corrupted state exceptions could mask errors (such as access violations), resulting in inconsistent state of execution or making it easier for attackers to compromise system. Instead, catch and handle a more specific set of exception type(s) or re-throw the exception."

This way the most relevant information Is shown and the user can expand the error in the error list to see the details.